generators/client/templates/angular/src/test/javascript/spec/app/shared/login/_login.component.spec.ts

```diff
@@ -16,8 +16,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -%>
-
-import { ComponentFixture, TestBed, async } from '@angular/core/testing';
+import { ComponentFixture, TestBed, async, inject, fakeAsync, tick } from '@angular/core/testing';
```

Comment:
does it require `tslint` disable ? 

Can we change it into multiline, tslint disable everywhere is not looking great

Reply:
Yes, its not worth the effort trying to make it multiline especially for the entity imports that are from ejs templates.